test/World_Test.java

```diff
@@ -0,0 +1,209 @@
+import com.laboon.World;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ * Created by Joshua Zwolan on 3/24/2015.
+ *
+ * NOTE: World.toString_OLD() is the old non-refactored method
+ */
+public class World_Test
+{
+    /*
+    -   Default Values
+    -   Used as Constants when the value is not being changed in Testing
+     */
+    final int SIZE =        50;
+    final int SEED =        9001;
+    final int PERCENT =     55;
+
+    /**
+     * The Following Tests the Default Values as defined Above
+     */
+    /*
+    -   Test to Make Sure Size of 50, Seed of 9001, and Percent of 55 Works
+    -   Uses Default Values for Size, Seed, and Percent
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_DEFAULT_VALUES()
+    {
+        World w = new World(SIZE, SEED, PERCENT);
+        assertEquals(w.toString_OLD(), w.toString());
```

Comment:
Good idea!  Going forward, though, we'd want to remove the _OLD method, so we couldn't do this.